CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Braintree iOS Visa Checkout SDK Release Notes
 
-This SDK component is currently in a limited release to [elligible merchants](https://articles.braintreepayments.com/guides/payment-methods/visa-checkout#limited-release-eligibility) and the API is subject to change.
+## unreleased
+
+* Update VisaCheckoutSDK to 7.2.0
+* Update minimum deployment target to iOS 12 and require Xcode 12
```

Comment:
Should these be listed under a Breaking Changes header?

Reply:
Good call - we can update that!